### DISCUSSION TITLE
Phazerville on After Later Audio

### DISCUSSION THREAD

**Discussion by ben-sec** [djphazer/O_C-Phazerville#70] [Q&A] [answered]
Hi!

What do I have to do to get Phazerville on my new After Later Audio OC? If I run Teensy on Windows with "o_C-phazerville-v1.7_phz-6d32547c.hex" and press the button on the Teensy board the module just turns off/black (yes, it's connected via USB). I saw tutorials that explain to me, that I have to connect the module first to USB and then power it on, but as soon as I connect it via USB it get's power and turns on. I'm confused ... Thanks in advance!

Cheers, Ben

**Comment by djphazer** [comment]
Looks like you've selected the correct file.
After Later does not sever the USB power trace on the Teensy, so you will see the display power on either way... This is normal. As long as the reflash is successful, you should be good to go!

If your PC cannot connect to the Teensy, the most common hiccup is using a micro USB charging cable (power only) instead of a proper data cable.

**Reply by ben-sec** [reply, marked as answer]
That was indeed the issue. I found another cable and that did the trick! Thanks a lot!
